examples/CMakeLists.txt

```diff
@@ -21,55 +21,62 @@ MACRO(BUILD_EXAMPLE EX_NAME EX_SRC COMPUTE_NAME FG_LIBS COMPUTE_LIBS)
 ENDMACRO()
 
 IF(TARGET forge)
+
     # Building with Forge Source
     SET(FORGE_FOUND False)
+    INCLUDE_DIRECTORIES(${GLBINDING_INCLUDE_DIRS})
+
 ELSE(TARGET forge)
+
     SET_PROPERTY(GLOBAL PROPERTY USE_FOLDERS ON)
-    FIND_PACKAGE(GLEWmx REQUIRED)
     FIND_PACKAGE(FreeImage REQUIRED)
```

Comment:
Is FreeImage required for compiling the examples too? Do we call any API directly?